src/lib/moduleMocker.js

```diff
@@ -47,6 +47,48 @@ function getType(ref) {
   return null;
 }
 
+function PlainConstructor() {}
+var plainObject = new PlainConstructor();
+/**
+ * methods on ES6 classes are not enumerable, so they can't
+ * be found with a simple `for (var slot in ...) {`
+ *
+ * scan will return an object with the same shape you would
+ * get with `for ... in` except the non-enumerable methods
+ * are included.
+ */
+ function scan(obj) {
```

Comment:
I'll still change it if you think `getMethods` is better, it just does more than that.